modules/nw-mutual-tls-auth.adoc

```diff
@@ -0,0 +1,55 @@
+// Module included in the following assemblies:
+//
+// * ingress/ingress-operator.adoc
+
+[id=nw-mutual-tls-auth_{context}]
+= Configuring Mutual TLS Authentication
+
+You can configure the Ingress Controller to enable mutual TLS (mTLS) authentication by setting a `spec.clientTLS` value. The `clientTLS` field specifies configuration for the Ingress Controller to verify client certificates. This configuration includes setting a `clientCA` value, which is a reference to a config map that contains the PEM-encoded CA certificate bundle that is used to verify a client's certificate, as well as an optional list of certificate subject filters.
+
+If the `clientCA` value specifies an X509v3 certificate revocation list (CRL) distribution point, the Ingress Operator downloads the CRL and configures the Ingress Controller to acknowledge it. Requests that do not provide valid certificates are rejected.
+
+.Prerequisites
+
+* You have access to the cluster as a user with the cluster-admin role.
+
+.Procedure
+Configure Mutual TLS authentication using the `spec.ClientTLS` field for the Ingress Controller.
+
+. Create a ConfigMap that is in the openshift-config namespace:
++
+[source,terminal]
+----
+$ oc create configmap router-ca-certs-default --from-file=ca-bundle.pem=client-ca.crt -n openshift-config
+----
++
+[NOTE]
+====
+The config map data key must follow a naming convention, such as `ca-bundle.pem`, and the data value must be a CA certificate in PEM format.
+====
+. Edit the `IngressController` resource in the `openshift-ingress-operator` project:
++
+[source,terminal]
+----
+$ oc edit IngressController default -n openshift-ingress-operator
+----
+
+.  Edit the yaml:
+
+.Sample `IngressController` CR for a `clientTLS` profile that specifies filtering patterns
+
```

Comment:
nit: The convention seems to be the block title, no blank line, and then the `[source,...]` line, like the first `.Example output` in the [Code blocks, command syntax, and example output](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#code-blocks-command-syntax-and-example-output) in the doc guidelines.